samples/client/petstore/php/OpenAPIClient-php/lib/Api/PetApi.php

```diff
@@ -558,7 +556,7 @@ protected function deletePetRequest($pet_id, $api_key = null)
      *
      * @throws \OpenAPI\Client\ApiException on non-2xx response
      * @throws \InvalidArgumentException
-     * @return \OpenAPI\Client\Model\Pet[]
+     * @return \OpenAPI\Client\Model\Pet[]|
```

Comment:
Unnecessary `|` have been added 🤔

Reply:
It is caused due to `{{responses}}` includes error responses (4xx, 5xx...) which has not `dataType`. 

It will fixed with using `{{#dataType}}` ~ `{{/dataType}}` like below 💡 :

```diff
      * @throws \{{invokerPackage}}\ApiException on non-2xx response
      * @throws \InvalidArgumentException
-     * @return array of {{#returnType}}{{#responses}}{{^-first}}|{{/-first}}{{{dataType}}}{{/responses}}{{/returnType}}{{^returnType}}null{{/returnType}}, HTTP status code, HTTP response headers (array of strings)
+     * @return array of {{#returnType}}{{#responses}}{{#dataType}}{{^-first}}|{{/-first}}{{{dataType}}}{{/dataType}}{{/responses}}{{/returnType}}{{^returnType}}null{{/returnType}}, HTTP status code, HTTP response headers (array of strings)
      */
     public function {{operationId}}WithHttpInfo({{#allParams}}${{paramName}}{{^required}} = {{#defaultValue}}{{{.}}}{{/defaultValue}}{{^defaultValue}}null{{/defaultValue}}{{/required}}{{#hasMore}}, {{/hasMore}}{{/allParams}})
     {
```

Reply:
Fixed :)